iaso/tests/api/test_profile_logs.py

```diff
@@ -0,0 +1,542 @@
+        self.assertEquals(data["count"], 2)
+        try:
+            jsonschema.validate(instance=data, schema=PROFILE_LOG_LIST_SCHEMA)
+        except jsonschema.exceptions.ValidationError as ex:
+            self.fail(msg=str(ex))
+
+        response = self.client.get(f"/api/userlogs/?user_ids={self.edited_user_1.id}&org_unit_id={self.org_unit_2.id}")
+        data = self.assertJSONResponse(response, 200)
+        self.assertEquals(data["count"], 2)
+        try:
+            jsonschema.validate(instance=data, schema=PROFILE_LOG_LIST_SCHEMA)
+        except jsonschema.exceptions.ValidationError as ex:
+            self.fail(msg=str(ex))
+
+        response = self.client.get(
+            f"/api/userlogs/?user_ids={self.edited_user_1.id}&modified_by={self.user_with_permission_1.id}&created_at_before=2020-02-14"
+        )
+        data = self.assertJSONResponse(response, 200)
+        self.assertEquals(data["count"], 1)
+        try:
+            jsonschema.validate(instance=data, schema=PROFILE_LOG_LIST_SCHEMA)
+        except jsonschema.exceptions.ValidationError as ex:
+            self.fail(msg=str(ex))
+
+        response = self.client.get(
+            f"/api/userlogs/?user_ids={self.edited_user_1.id}&modified_by={self.user_with_permission_1.id}&created_at_after=2020-02-14"
+        )
+        data = self.assertJSONResponse(response, 200)
+        self.assertEquals(data["count"], 1)
+        try:
+            jsonschema.validate(instance=data, schema=PROFILE_LOG_LIST_SCHEMA)
+        except jsonschema.exceptions.ValidationError as ex:
+            self.fail(msg=str(ex))
+        results = data["results"]
+        self.assertEquals(len(results), 1)
+        user = results[0]["user"]
+        self.assertEquals(user["id"], self.edited_user_1.iaso_profile.id)
+        self.assertEquals(user["user_id"], self.edited_user_1.id)
+        self.assertEquals(user["username"], self.edited_user_1.username)
+        self.assertEquals(user["first_name"], self.edited_user_1.first_name)
+        self.assertEquals(user["last_name"], self.edited_user_1.last_name)
+        modified_by = results[0]["modified_by"]
+        self.assertEquals(modified_by["id"], self.user_with_permission_1.iaso_profile.id)
+        self.assertEquals(modified_by["user_id"], self.user_with_permission_1.id)
+        self.assertEquals(modified_by["username"], self.user_with_permission_1.username)
+        self.assertEquals(modified_by["first_name"], self.user_with_permission_1.first_name)
+        self.assertEquals(modified_by["last_name"], self.user_with_permission_1.last_name)
+        past_location = results[0]["past_location"][0]
+        self.assertEquals(past_location["name"], self.org_unit_1.name)
+        self.assertEquals(past_location["id"], self.org_unit_1.id)
+        new_location = results[0]["new_location"][0]
+        self.assertEquals(new_location["name"], self.org_unit_1.name)
+        self.assertEquals(new_location["id"], self.org_unit_1.id)
```

Comment:
I maybe would have split this part into another test, for testing the content of what is returned